content/enterprise_influxdb/v1.9/concepts/schema_and_data_layout.md

```diff
@@ -9,136 +9,160 @@ menu:
+High series cardinality is a primary driver of high memory usage for many database workloads.
+Therefore, to reduce memory consumption, consider storing high-cardinality values in fields values rather than in tags or field keys.
 
-#### Avoid using keywords as tag or field names
+{{% note %}}
 
-Not required, but simplifies writing queries because you won't have to wrap tag or field names in double quotes.
-See [InfluxQL](https://github.com/influxdata/influxql/blob/master/README.md#keywords) and [Flux](https://github.com/influxdata/flux/blob/master/docs/SPEC.md#keywords) keywords to avoid.
+If reads and writes to InfluxDB start to slow down, you may have high series cardinality (too many series).
+See [how to find and reduce high series cardinality](/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions/#why-does-series-cardinality-matter).
 
-Also, if a tag or field name contains characters other than `[A-z,_]`, you must wrap it in double quotes in InfluxQL or use [bracket notation](/{{< latest "influxdb" "v2" >}}/query-data/get-started/syntax-basics/#records) in Flux.
+{{% /note %}}
 
-### Discouraged schema design
+## Use recommended naming conventions
 
-We recommend that you:
+Use the following conventions when naming your tag and field keys:
 
-- [Avoid too many series](#avoid-too-many-series)
-- [Avoid the same name for a tag and a field](#avoid-the-same-name-for-a-tag-and-a-field)
-- [Avoid encoding data in measurement names](#avoid-encoding-data-in-measurement-names)
-- [Avoid putting more than one piece of information in one tag](#avoid-putting-more-than-one-piece-of-information-in-one-tag)
+- [Avoid reserved keywords in tag and field keys](#avoid-reserved-keywords-in-tag-and-field-keys)
+- [Avoid the same tag and field name](#avoid-the-same-name-for-a-tag-and-a-field)
+- [Avoid encoding data in measurements and keys](#avoid-encoding-data-in-measurements-and-keys)
+- [Avoid more than one piece of information in one tag](#avoid-putting-more-than-one-piece-of-information-in-one-tag)
 
-#### Avoid too many series
+### Avoid reserved keywords in tag and field keys
 
-[Tags](/enterprise_influxdb/v1.9/concepts/glossary/#tag) containing highly variable information like UUIDs, hashes, and random strings lead to a large number of [series](/enterprise_influxdb/v1.9/concepts/glossary/#series) in the database, also known as high series cardinality. High series cardinality is a primary driver of high memory usage for many database workloads.
+Not required, but avoiding the use of reserved keywords in your tag and field keys simplifies writing queries because you won't have to wrap your keys in double quotes.
```

Comment:
```suggestion
Not required, but avoiding the use of reserved keywords in your tag keys and field keys simplifies writing queries because you won't have to wrap your keys in double quotes.
```